src/WebJobs.Extensions.DurableTask/HttpApiHandler.cs

```diff
@@ -243,7 +261,7 @@ public HttpApiHandler(DurableTaskExtension config, TraceWriter traceWriter)
             }
 
             string mediaType = request.Content.Headers.ContentType?.MediaType;
-            if (!string.IsNullOrEmpty(mediaType) && 
+            if (!string.IsNullOrEmpty(mediaType) &&
```

Comment:
Isn't this check redundant? `string.Equals` knows how to handle null values.

Reply:
Good point